server/src/transport/ws.rs

```diff
@@ -379,7 +380,8 @@ async fn send_task(
 			}
 
 			// Handle timer intervals.
-			Either::Right((Either::Left((Some(_instant), stop)), next_rx)) => {
+			Either::Right((Either::Left((_instant, _stopped)), next_rx)) => {
+				stop = _stopped;
```

Comment:
nit: Are we adding this line for better readability?

Reply:
no, ideally it should be removed :)